src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -780,37 +919,54 @@ default long realCapacity() {
     }
 
     /**
-     * @return a copy of this Bytes from position() to limit().
+     * @return a copy of this Bytes object from position() to limit().
      */
     @Override
-    BytesStore<Bytes<Underlying>, Underlying> copy()
+    BytesStore<Bytes<U>, U> copy()
             throws IllegalStateException;
 
+    /**
+     * Creates and returns a new String representing the contents of this Bytes object in hexadecimal form encoding
+     * at most 1024 bytes.
+     *
+     * @return a new String representing the contents of this Bytes object in hexadecimal form encoding
+     * at most 1024 bytes
+     */
     @NotNull
     default String toHexString() {
         return toHexString(1024);
     }
 
     /**
+     * Creates and returns a new String representing the contents of this Bytes object in hexadecimal form encoding
+     * at most the provided {@code maxLength } bytes.
```

Comment:
Again this description is describing parameters which are described in `@param` I wonder if we should attempt to keep the headline descriptions brief and rely on the other javadoc to tell the detailed story?

Reply:
(I'm very pro keeping documentation as short an normalised as possible, in the interest of minimising the opportunity for them to go stale)

Reply:
I think the headline should mention the parameters and how they broadly affect the method and then the `@param` could be more detailed (e.g. non-null, no side effect, stateless...) info. I could try to trim off the headlines a bit and try to move info into the `@param` tags.